src/raygui.h

```diff
@@ -2896,68 +2896,72 @@ RAYGUIDEF void GuiLoadStyle(const char *fileName)
     if (rgsFile != NULL)
     {
         char buffer[256] = { 0 };
-        fgets(buffer, 256, rgsFile);
-
-        if (buffer[0] == '#')
+        if ( fgets(buffer, 256, rgsFile) != NULL )
```

Comment:
fgets will return nullptr on error.
[https://www.tutorialspoint.com/c_standard_library/c_function_fgets.htm](url)

Reply:
This looks like I did more changes than required, just tabbed into the if statement.